src/javascript/app/pages/trade/tick_trade.js

```diff
@@ -231,7 +231,7 @@ const TickDisplay = (() => {
                 value: calc_barrier,
                 color: 'green',
                 label: {
-                    text : `Average (${addComma(calc_barrier)})`,
+                    text : `${localize('Average')} (${addComma(calc_barrier)})`,
```

Comment:
I think we don't have 'Average' in generate static data, I had to add it in my other PR. also, do we have any more `Barrier ([_1])` (high barrier etc variations as well) anywhere? if no, we can remove them from generate static data

Reply:
you already added it in your hi_lo PR: https://github.com/binary-com/binary-static/pull/3845/files#diff-4a55e57599e7d49fa3a6bdfedc88160bR241 :) 
and yeah, i missed out some `Barrier ([_1]) ` strings, will submit a fix here

Reply:
yeah, but I don't know if my PR will get merged before yours or not :(

Reply:
it's okay, we still need to localize `Average`, no? :P